language/de-DE/joomla.ini

```
@@ -108,6 +108,7 @@ JSAVE="Speichern"
 JSELECT="Auswählen"
 JSHOW="Anzeigen"
 JSITE="Site"
+JSITEADMIN="Client wählen"
```

```suggestion
JSITEADMIN="Bereich wählen"
```
Hört sich doch besser an als Client, oder?